### ISSUE TITLE
owl-carusel not handling having the slug set

### ISSUE THREAD

**Issue by Athanasiamo** [victoriadrake/hugo-theme-introduction#206]
Hi. 

On my page I specifically set the slug of posts to be prefixed by post dates, because that makes content easier to keep track of over time.
The Owl carousel unfortunately does not seem to handle the slug being set like that. 
While the title, image and date seems to be correct in the carousel, the link to the post is not.

for instance, I forgot to set the slug as I would usually in my newest post, so the link to that works. But all other posts, where I have the date appended, it does not work.

https://drmowinckels.io/blog/2020-04-30-using-freesurfer-annotation-files-to-plot-in-r/

**Comment by victoriadrake** [comment]
I'm afraid I don't see what you're describing. I don't see a difference between your newest post and the one you linked.

The carousel code in this theme is currently used on pages in the projects section. Do you mean something different, perhaps?

**Comment by Athanasiamo** [comment]
Maybe I'm misreading the code somewhere. I mean at the bottom of a post, before comments section, there is a carousel for other "related" posts. These do not correctly link to posts where slugs are set differently from default.

![image](https://user-images.githubusercontent.com/14014329/83360414-807ce980-a381-11ea-8ee2-367e11f6c5bd.png)

**Comment by Athanasiamo** [comment]
Hi.

After quite some digging, I found that this carousel comes from the diqsus comment section. 
This is a new feature they have they call "Recommendations", and it does not handle url changes well. I had to make a csv mapping old and new urls after bundling my posts:
https://help.disqus.com/en/articles/1717068-migration-tools

The recommendations can also be turned off in the disqus admin panel. 

So this has nothing to do with the theme, sorry about the confusion. I hope if someone encounters something similar they might find this comment and get it sorted.
